### DISCUSSION TITLE
"Hidden" notifications

### DISCUSSION THREAD

**Discussion by joemasilotti** [excid3/noticed#273] [Q&A] [answered]
My goal is to trigger a Noticed notification that only sends an email and _does not_ show up in the UI for a user.

Is there anything built in to Noticed to handle this? If not, should I be overwriting `format: :format_for_database` to add a e.g. `visible: false` key then scoping in the UI?

Or am I overthinking it and should just be interacting with a mailer directly?

[More context](https://github.com/joemasilotti/railsdevs.com/issues/789).

**Comment by excid3** [comment, marked as answer]
Remove the database delivery method.

**Reply by joemasilotti** [reply]
But then there's no "record" of the notification being sent, right? It gets fired and forgotten?

What if I want to delay the sending of the email?

**Reply by excid3** [reply]
That's how most emails work, you don't usually store it in the db.

Add a delay to the notification or to the email if you want to delay it.

**Reply by joemasilotti** [reply]
Fair enough. Thanks Chris!
